apis/apps/v1alpha1/nebulabackup_types.go

```diff
@@ -98,16 +98,22 @@ type BackupSpec struct {
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
 	BR *BRConfig `json:"br,omitempty"`
+
+	// NumBackupsKeep is to specify how many backups we want to keep. Used for scheduled backups only.
+	NumBackupsKeep *int32 `json:"maxBackups,omitempty"`
+
+	// ReservedTimeEpoach is the epoach time for how long we want to keep the backups. Used for scheduled backups only.
+	ReservedTimeEpoch *time.Duration `json:"maxReservedTimeEpoch,omitempty"`
```

Comment:
These two fields used for nsb only, I think you can use EnvVar replace them. Then merge the EnvVar in func generateBackupJob.

Reply:
Fixed